src/vs/base/browser/ui/tree/asyncDataTree.ts

```diff
@@ -968,7 +758,13 @@ export class AsyncDataTree<TInput, T, TFilterData = void> implements IDisposable
 			await Event.toPromise(this._onDidRender.event);
 		}
 
-		const node = this.getDataNode(element);
+		let node;
+		try {
+			node = this.getDataNode(element);
+		} catch {
+			console.log('Error in _updateChildren');
```

Comment:
This is a bad idea. If this happens, the tree will be in a very bad state and there's no recovering from here. So we want the error to be thrown instead of quietly logging and pretending everything is fine.

Reply:
If this indeed is something that is happening, we must dig into the root cause and fix the issue upstream.

Reply:
Nice catch, I actually forgot to remove it.